media/stitcher/api/Stitcher.Samples/UpdateCdnKey.cs

```diff
@@ -0,0 +1,74 @@
+﻿/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START video_stitcher_update_cdn_key]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Video.Stitcher.V1;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+
+public class UpdateCdnKeySample
+{
+    public CdnKey UpdateCdnKey(
+        string projectId, string location, string cdnKeyId, string hostname,
+        string gcdnKeyname, string gcdnPrivateKey, string akamaiTokenKey)
+    {
+        // Create the client.
+        VideoStitcherServiceClient client = VideoStitcherServiceClient.Create();
+
+        // Build the parent location name.
+        LocationName parentLocation = new LocationName(projectId, location);
+
+        CdnKey cdnKey = new CdnKey
+        {
+            CdnKeyName = CdnKeyName.FromProjectLocationCdnKey(projectId, location, cdnKeyId),
+            Hostname = hostname
+        };
+
+        string path;
+        if (akamaiTokenKey != "")

```

Comment:
It would be more idiomatic to pass a null value than an empty string, so the check would be (and invert the code within the if-else clauses.)
```suggestion
        if (akamaiTokenKey is null)

```

Reply:
Done